### ISSUE TITLE
[design] CST status indicator inaccessible to colorblind veteran

### ISSUE THREAD

**Issue by sortizsh** [department-of-veterans-affairs/va.gov-team#53016]
**Issue:** 
Veteran with colorblindness is unable to determine their claim status based upon what the circle is indicating (see attached screeenshot).  Veteran was also confused since it appeared as if the claim was granted or denied since they could not tell the difference between the green and red fill on the status circle. 

**Device:** 
phone, unknown if VA app or VA.gov in browser 

![unnamed.jpg](https://images.zenhubusercontent.com/5b1e83d2a17bb42738f85a42/eecdf943-c8ae-4cff-960a-50afd0820d42)

**Comment by sortizsh** [comment]
Probably need @skylerschain  and @joshkimux  to take a look

**Comment by sortizsh** [comment]
@jacobworrell  for next grooming session

**Comment by jacobworrell** [comment]
I will ping Samara again on this question.

**Comment by jaredcunha** [comment]
Does the dot indicate Granted or Current Step? This could be a content issue.

**Comment by HeatherWidmont** [comment]
@Samara-Strauss please refer to Jared's question above and provide an answer when you can. Thank you!

**Comment by HeatherWidmont** [comment]
We will check back in with Samara after she returns on July 10th.

**Comment by HeatherWidmont** [comment]
Heather will reach out to Devon, the original requester, for the veteran's claim/appeal ID & a full screenshot in order to reproduce the issue.

**Comment by HeatherWidmont** [comment]
I haven't seen a response and since this problem is being addressed in the new designs we are going to close this ticket

**Comment by Samara-Strauss** [comment]
Hi all -- I'm sorry for the delayed response. I was on maternity leave and just returned today. Did you all get an answer to your question? I haven't worked on the claim status tool, so I'm not sure if I can help, but I can probably help direct this question if you still need answers.

**Comment by HeatherWidmont** [comment]
Hello, congratulations!! We weren't able to reproduce the issue without the veteran's ID & it's already being addressed in the new designs so we decided to close the ticket. We no longer need any answers, thanks @Samara-Strauss!
